sandy/core/cov.py

```diff
@@ -2208,18 +2222,35 @@ def triu_matrix(matrix, kind='upper'):
     Examples
     --------
     >>> S = pd.DataFrame(np.array([[1, 2, 1], [0, 2, 4], [0, 0, 3]]))
-    >>> triu_matrix(S)
-    	0	1	2
-    0	1	2	1
-    1	2	2	4
-    2	1	4	3
+    >>> triu_matrix(S).data
+                0           1           2
+    0 1.00000e+00 2.00000e+00 1.00000e+00
+    1 2.00000e+00 2.00000e+00 4.00000e+00
+    2 1.00000e+00 4.00000e+00 3.00000e+00
 
+    Suppose that upper triangular matrix part is symmetric parcially symmetric:

```

Comment:
what does 'symmetric parcially symmetric' mean? Partial symmetry is not really a thing, right?!
Also, it is 'partially', not 'parcially'

Reply:
if you want to show that the lower part will be overwritten, then I suggest you use more different values, e.g.,
S = pd.DataFrame(np.array([[1, 2, 1], [-8, 2, 4], [-6, -5, 3]]))